samples/pet-store/components/catalog/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "pet-store-catalog",
+  "version": "1.0.0",
+  "description": "package.json file for catalog of pet-store sample",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wso2-cellery/sdk.git"
+  },
+  "keywords": [
+    "catalog",
+    "pet-store",
+    "cellery"
+  ],
+  "author": "asmaj@wso2.com",
+  "license": "ISC",
```

Comment:
Shall we change this ? The license and author ? We can do this for all the package.json files